docs/source/userguide.rst

```diff
@@ -245,16 +156,50 @@ The steps for training a model with HPO are identical to the default :code:`Simp
     )
 
     # fit an imputer model with customized hyperparameters
-    imputer.fit_hpo(
-        train_df=df_train,
-        num_epochs=100,
-        patience=3,
-        learning_rate_candidates=[1e-3, 3e-4, 1e-4]
-    )
+    imputer.fit_hpo(train_df=df_train)
+
+Calling HPO like this will search through some basic and usually helpful hyperparameter choices.
+There are two ways for a more detailed search. Firstly, :code:`fit_hpo` offers additional arguments that can be inspected in the SimpleImputer_. For even more configurations and variation of hyperparameters for the various input column types, a dictionary with ranges can be passed to :code:`fit_hpo` as can be seen in the hpo-code_.
```

Comment:
`hpo-code` should point to the public entrypoint of the hyper params dict here: https://github.com/awslabs/datawig/blob/c0d25631e9cda567577d2ed5a34089716831a565/datawig/simple_imputer.py#L167.